working/augmentation-libraries/feature-specification.md

```diff
@@ -163,78 +182,72 @@ Augmentation declarations include:
-declarations. When that happens, the *augmentation application order* defines
-in which order the augmentations are applied, with later augmentations applying
-to the result of applying all earlier augmentations to the original base
-declaration. The augmentation application order is defined rather simply,
-because of a further requirement on where augmentations of the same declaration
-can occur relative to each other.
-
-It’s a **compile-time error** if a library contains an augmentation
-declaration, but no non-augmentation declaration with the corresponding name in
-the same scope. _(A mutable variable declaration counts as having both a getter
-and a setter name.)_
-
-For the following, we’ll say that one declaration of a library is *above*
-another declaration of the same library if and only if:
-
-*   The former declaration is in the same file as the latter declaration, and it
-    is textually earlier in the file (“above” in the source code as normally
-    presented), or
-*   The former declaration is in a file that is a direct or transitive parent
-    file of the file of the latter declaration (“above” in the file tree
-    hierarchy).
-
-We can similarly define *below* as the inverse of that relation. Both *before*
-and *after* define *strict partial orders* on declarations in a library.
-
-It’s a **compile-time error** if a library contains an augmentation declaration
-and a corresponding non-augmentation base declaration, and the base
-declaration is not *above* the augmentation declaration.
-
-These requirements ensure that declarations that contribute to the same
-effective declaration, one base declaration and zero or more augmentation
-declarations, are *totally ordered* by the *above* relation, with the base
-declaration at the top, and the declarations all being in files on a single
-*path* down the file tree.
-
-The *augmentation application order* for a single base declaration’s (validly
-positioned) augmentation declarations is then in *before* order: An augmentation
-declaration is applied after any augmentation declarations that are *before* it,
-and before augmentation declarations that it is before.
-
-This applies both to top-level declarations and to member declarations of, for
-example, class declarations.
+declarations. This occurs in the situation where an augmentation
+declaration has an augmented declaration which is itself an augmentation
+declaration, and so on, until an introductory declaration is reached. This
+will happen in a finite number of steps because of the nature of the
+before/after ordering.
+
+Declarations that contribute to the same effective declaration, one
+introductory declaration and zero or more augmentation declarations with
+the same name and in the same context, are *totally ordered* by the *after*
+relation, with the introductory declaration being least, and the
+augmentation declarations greater than that and totally ordered.
```

Comment:
Nit: Remove "and totally ordered" since you already said that.

Reply:
Done.